packages/features/ee/organizations/pages/settings/other-team-members-view.tsx

```diff
@@ -192,7 +192,12 @@ const MembersView = () => {
             {team && (
               <>
                 <hr className="border-subtle my-8" />
-                <MakeTeamPrivateSwitch teamId={team.id} isPrivate={team.isPrivate} disabled={false} />
+                <MakeTeamPrivateSwitch
+                  teamId={team.id}
+                  isPrivate={team.isPrivate}
+                  disabled={false}
+                  isOrg={false}
```

Comment:
"isOrg" was missing from this "MakeTeamPrivateSwitch" so I added it. Even though the "MembersView" component in this file is in "ee/organizations" pages, my guess is that "isOrg" has to be "false" because up until now we had the private functionality only for teams and this code was already here, and the "<Meta" of "MembersView" uses "team_members" and "members_team_description" as keys.

Reply:
Yeah, didn't notice this. Thanks!